test/TesterAzureUtils/Persistence/PersistenceGrainTests_AzureTableGrainStorage.cs

```diff
@@ -29,8 +28,6 @@ protected override void ConfigureTestCluster(TestClusterBuilder builder)
                 {
                     legacy.ClusterConfiguration.Globals.DataConnectionString = TestDefaultConfiguration.DataConnectionString;
 
-                    legacy.ClusterConfiguration.Globals.ServiceId = ServiceId;
```

Comment:
Why id you remove this?

Reply:
Technically, the tests work without it - with the default empty GUID `ServiceId`. I pushed a couple of commits to set it to a new GUID to avoid that.